testnet/stacks-node/src/main.rs

```diff
@@ -77,6 +77,13 @@ fn main() {
     let mut args = Arguments::from_env();
     let subcommand = args.subcommand().unwrap().unwrap_or_default();
 
+    let server_version = stacks::version_string(
+        option_env!("CARGO_PKG_NAME").unwrap_or("stacks-node"),
+        option_env!("CARGO_PKG_VERSION").unwrap_or("0.0.0.0"),
```

Comment:
I believe the way we generate the `version_string` changed a little bit, we use:

```rust
                &stacks::version_string(
                    "stacks-node",
                    option_env!("STACKS_NODE_VERSION")
                        .or(option_env!("CARGO_PKG_VERSION"))
                        .unwrap_or("0.0.0.0")
                )
```

in the `--version` output in this file.

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/2234/commits/445e25c5863951ce0606d59b4cf66b08529bf801